File: JavaScript/Codewars/6 kyu/multi_of_3_or_5.js
// CODEWARS.COM - Multiples of 3 or 5
function solution(number) {
    let result = 0;
    for (let num = (number - 1); num > 0; num--) {
        if (num % 3 == 0 || num % 5 == 0) {
            result += num;
        }
    }
    return result;
    // console.log(result);
}

module.exports = solution;

File: C#/Codewars/6kyu/multi_of_3_or_5.cs
public static class Kata
{
    public static int Solution(int value)
    {
        int result = 0;
        for (int num = (value - 1); num > 0; num--)
        {
            if (num % 3 == 0 || num % 5 == 0)
            {
                result += num;
            }
        }
        return result;
    }
}